build-and-test-all-mysql-maven.sh

```diff
@@ -13,11 +13,11 @@ docker-compose -f docker-compose-mysql-binlog-maven.yml up --build -d mysql zook
 
 docker-compose -f docker-compose-mysql-binlog-maven.yml up --build -d cdc-service
 
-./wait-for-services.sh ${DOCKER_HOST_IP:-localhost} "8099"
+./wait-for-services.sh ${DOCKER_HOST_IP:-localhost} "actuator/health" "8099"
 
 docker-compose -f docker-compose-mysql-binlog-maven.yml up --build -d
 
-./wait-for-services.sh ${DOCKER_HOST_IP:-localhost} "8081 8082 8083"
+./wait-for-services.sh ${DOCKER_HOST_IP:-localhost} "actuator/health" "8081 8082 8083"
```

Comment:
ports are `$*` so there shouldn't be quote around the port numbers.

Reply:
ok, I will remove, but with quotes it also works in the same way too.